### PR TITLE
hbase-store: clean-ups

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: store/hbase/kv.go [hunk 1]
```diff
@@ -25,14 +24,14 @@ import (
...
-	// ColFamily is the hbase column family name.
-	ColFamily = "f"
-	// Qualifier is the hbase column name.
-	Qualifier = "q"
-	// FmlAndQual is a shortcut.
-	FmlAndQual = ColFamily + ":" + Qualifier
-	// HbaseTableName is the table name in hbase which is invisible to SQL layer.
-	HbaseTableName = "tidb"
+	// hbaseColFamily is the hbase column family name.
+	hbaseColFamily = "f"
```
Comment (ngaut) [inline store/hbase/kv.go]: You may create both  hbaseColFamilyStr and hbaseColFamilyBytes, so you don't have to convert it in Get method.

File: store/hbase/snapshot.go [hunk 2]
```diff
@@ -140,7 +121,7 @@ func internalGet(s *hbaseSnapshot, g *hbase.Get) ([]byte, error) {
 	if r == nil || len(r.Columns) == 0 {
 		return nil, kv.ErrNotExist
```
Comment (siddontang) [inline store/hbase/snapshot.go]: errors.Trace

File: store/hbase/snapshot.go [hunk 3]
```diff
@@ -89,9 +72,8 @@ func (s *hbaseSnapshot) BatchGet(keys []kv.Key) (map[string][]byte, error) {
 	m := make(map[string][]byte)
```
Comment (siddontang) [inline store/hbase/snapshot.go]: pre-alloc m with len(rows) capability.

**Comment by ngaut** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**